README.md

```diff
@@ -81,6 +84,26 @@ Make as much of this opensource-ready as possible:
 - Tabs component
 
 
+## How to Structure Modules for A Component
+
+We use "component" here in its loosest sense: ranging from a reusable view function to a pluggable trio of model/view/update functions.
+
+When the component is small enough, it's fine to let a single module hold all relevant code:
+
+- Nri/
+  - Button.elm
+
+When the component gets more complex, break out each of the Elm architecture triad into its own file:
```

Comment:
do you have guidelines you use for this? In my case it's a gut decision. If it's the same for you, maybe we should call that out.

Reply:
Nope, gut decision as well. Called out in 11c26fa5d8b10d28e1ec463c8eeea5fe70dbde20